aptos-move/framework/aptos-framework/sources/Coin.move

```diff
@@ -149,12 +163,14 @@ module AptosFramework::Coin {
         Coin { value: amount }
     }
 
-    public(script) fun initialize<CoinType>(
```

Comment:
Well, initial idea we discussed is to seperate scripts and public functions in different modules (Coin and ManagedCoin). 100% we need non-script version, otherwise it's just impossible to mint new coins for 3rd party modules for example, and many other things.